admin/configuration/culling.md

```diff
@@ -0,0 +1,11 @@
+# User server culling
+
+To ensure efficient resource usage, user servers without interactive usage for a
+period of time (default 1h) are automatically stopped (via
+[jupyterhub-idle-culler](https://github.com/jupyterhub/jupyterhub-idle-culler)).
+This means your notebook server might be stopped for inactivity even if you have
+a long running process in the notebook. This timeout can be configured.
```

Comment:
We should link to the actual way to configure it, no?

Reply:
Yeah, but that requires we add more docs in the pilot-hubs repo for SREs, which doesn't exist atm. I'll add a TODO here